src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -344,26 +344,21 @@ private BoundExpression CreateTupleConversion(CSharpSyntaxNode syntax, BoundTupl
 
             NamedTypeSymbol targetType = (NamedTypeSymbol)destinationWithoutNullable;
 
-            NamedTypeSymbol targetUnderlyingType;
             if (targetType.IsTupleType)
             {
                 var destTupleType = (TupleTypeSymbol)targetType;
-                targetUnderlyingType = destTupleType.UnderlyingTupleType;
                 // do not lose the original element names in the literal if different from names in the target
+
+                // PROTOTYPE(tuples): Come back to this, what about locations?
                 targetType = destTupleType.WithElementNames(sourceTuple.ArgumentNamesOpt);
```

Comment:
Resolved by recording open issue [https://github.com/dotnet/roslyn/issues/11013](https://github.com/dotnet/roslyn/issues/11013)

---
In reply to: [61779682](https://github.com/dotnet/roslyn/pull/11006#discussion_r61779682) [](ancestors = 61779682)